### PR TITLE
fix: merge latest beta into next 23jan

### PR DESCRIPTION
**Pull request by mkesavan13**
# COMPLETES Adhoc PR

## This pull request addresses

Merging of latest beta code into the next branch

## by making the following changes

- By doing a git merge for changes before January 23, 2024
- And Cherry-picking all the other changes from January 23, 2024

## The following scenarios were tested

1. Meeting initialize and register
2. Create and sync meetings
3. Stream management (Applying VBG for all three cases and BNR for Audio)
4. Screen Share

All of the above were tested in both Single and Multistream using a Personal Access Token

### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/internal-plugin-dss/src/dss.ts [hunk 1]
```diff
@@ -412,31 +412,6 @@ const DSS = WebexPlugin.extend({
...
   },
-
-  /**
-   * Search for information about places
-   * @param {Object} options
-   * @param {string} options.queryString A query string that will be transformed into a Directory search filter query. It is used to search the following fields: placeName, displayName.
-   * @param {number} options.resultSize The maximum number of results returned from each provider
-   * @returns {Promise} Resolves with an array of entities found
-   */
-  searchPlaces(options: SearchPlaceOptions) {
```
Comment (mkesavan13) [inline packages/@webex/internal-plugin-dss/src/dss.ts:423]: Removed this method because it was repeating. Expand it at the top to know

File: README.md [hunk 2]
```diff
@@ -175,7 +175,7 @@ webex.rooms
 
 ## Samples
 
-Sample code can be found in [docs/samples](./docs/samples). You can demo them by going to [webex.github.io/webex-js-sdk/samples/](https://webex.github.io/webex-js-sdk/samples/) or you can run them yourself by following this guide in the [Contribution Guide](CONTRIBUTING.md#running-samples-locally)
+Sample code can be found in [docs/samples](./docs/samples). You can demo them by going to [webex.github.io/webex-js-sdk/samples/](https://webex.github.io/webex-js-sdk/samples/browser-plugin-meetings/) or you can run them yourself by following this guide in the [Contribution Guide](CONTRIBUTING.md#running-samples-locally)
```
Comment (sreenara) [inline README.md:178]: Why this change? I don't think we need this.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,104 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [2.59.0](https://github.com/webex/webex-js-sdk/compare/v2.58.1...v2.59.0) (2023-07-24)
+## [3.0.0-beta.318](https://github.com/webex/webex-js-sdk/compare/v3.0.0-beta.317...v3.0.0-beta.318) (2024-01-19)
```
Comment (sreenara) [inline CHANGELOG.md:5]: Do we need the beta changelogs in the `next` branch?

**Review by sreenara — CHANGES_REQUESTED** [review]
Please see the comments

File: tsconfig.json [hunk 4]
```diff
@@ -20,6 +20,7 @@
     "sourceMap": true,
     "stripInternal": true,
     "target": "ES6",
+    "skipLibCheck": true,
```
Comment (sreenara) [inline tsconfig.json:23]: This is a duplicate of line 19. It's causing the build to fail.

**Review by sreenara — CHANGES_REQUESTED** [review]
Please fix the issue in tsconfig.json which is breaking the build

File: docs/api/index.html [hunk 5]
```diff
@@ -37751,6 +37752,287 @@ <h3 class='fl m0' id='webexmeetingserror'>
...
+  
+
+  
+
+  
+
+  
+
+  
+</section>
```
[the remark this answers is not in the document]
Comment (mkesavan13) [reply docs/api/index.html:37769]: This doesn't affect the KS app. It's the index page of API Reference and it is generated every time a merge happens to the next and goes into the documentation branch. So we need not worry about this.

File: README.md (hunk 2 above)
Comment (mkesavan13) [reply README.md]: Changed it back. This change came in because, beta made that edit.

File: CHANGELOG.md (hunk 3 above)
Comment (mkesavan13) [reply CHANGELOG.md]: Mistakenly included beta changes instead of next. Changed now.

**Review by sreenara — APPROVED** [review]
Please list the sanity tests in the description and we can merge
